src/core/src/MatrixDynSize.cpp

```diff
@@ -42,6 +42,29 @@ MatrixDynSize::MatrixDynSize(unsigned int _rows,
     zero();
 }
 
+MatrixDynSize::MatrixDynSize(const MatrixView<const double>& other) : m_rows(other.rows()),
```

Comment:
I am a bit rusty in iDynTree.. is the ordering defined only for the sparse matrix?
Are dense matrices always row major?

Reply:
Yes, dense matrices are always row major.